packages/docs/modules/ROOT/pages/public_deploy.adoc

```diff
@@ -0,0 +1,185 @@
+[[deploy-your-contracts-to-a-public-network]]
+= Deploy your contracts to a public network
+
+This tutorial link:first[builds on the first tutorial] where we created a new OpenZeppelin SDK project, and created a simple `Counter` contract in a local development network. We will now see how to deploy this contract to a public network.  
+
+We will use the Ropsten public test network for this tutorial.  You could also use other public test networks such as Rinkeby, Kovan or Goerli.
```

Comment:
Let's add a note block here briefly explaining the differences between the testnets. At least, let's point out that ropsten is PoW while the others are PoA (and what that implies in terms of block times), and that Rinkeby and Kovan only work with Geth and Parity respectively.

Reply:
Added, though not sure how to add new lines in a Note.

Reply:
Added